src/buttons/Button.js

```diff
@@ -56,6 +56,16 @@ class Button extends Component {
       ...attributes
     } = this.props;
 
+    if (Platform.OS === 'android' && (borderRadius && !attributes.background)) {
```

Comment:
`borderRadius` doesn't exist here. You need to use `containerStyle.borderRadius`

Reply:
when you rely on linters too much XD